bin/entrypoint

```diff
@@ -227,6 +227,24 @@ function fallback_admin() {
   fi
 }
 
+function aio_clamav () {
+  if [ -z "$ENSURE_AIO_CLAMAV_CONFIG" ]; then
+    echo 'ENSURE_AIO_CLAMAV_CONFIG="ON"' >> "$STW_AIO_ENV"
+    ENSURE_AIO_CLAMAV_CONFIG="ON"
+  fi
+
+  if [ "$ENSURE_AIO_CLAMAV_CONFIG" = "ON" ] && [ "$CLAMAV_ENABLED" = "yes" ]; then
```

Comment:
LGTM together with https://github.com/nextcloud/all-in-one/pull/6770. Unfortunately I cannot say too much about the rest of the PR...